reframe/core/modules.py

```diff
@@ -997,18 +985,12 @@ def available_modules(self, substr):
         return ret
 
     def _module_full_name(self, name):
-        return self.execute('find', '--format', self._name_format,
-                            name)
+        return self.execute('find', '--format', self._name_format, name)
 
     def loaded_modules(self):
```

Comment:
This function is untested for the Spack backend...

Reply:
Initially `_module_full_name` was needed several times but it ended up being used only once. Maybe we get rid of it?

Reply:
Yes, we can get rid of it.